GUI/AllegroInput.h

```diff
@@ -14,9 +14,9 @@
 
 
 enum aMButtons {
-    AMBLEFT = 1,
-    AMBRIGHT = 2,
-    AMBMIDDLE = 4
+    AMBLEFT = 1, // 0b1
```

Comment:
These comments are to explain that these entries are used as bit values right? I don't know if they're really necessary, but maybe they're helpful for people who'd have to look it up to doublecheck what the numbers correspond to.

Reply:
right.